rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -636,7 +681,7 @@ s32 cellGemGetMemorySize(s32 max_connect)
 	return max_connect <= 2 ? 0x120000 : 0x140000;
```

Comment:
same here and all the other that don't return CELL_OK

Reply:
I totally missed that one. thx